sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs

```diff
@@ -1,205 +1,96 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
 using System.Collections.Generic;
-using System.Globalization;
-using System.Linq;
-using System.Text;
+using System.ComponentModel;
 using Azure.Core.TestFramework.Models;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 
 namespace Azure.Core.TestFramework
 {
     public class RecordedTestSanitizer
     {
         public const string SanitizeValue = "Sanitized";
-        internal List<(string JsonPath, Func<JToken, JToken> Sanitizer)> JsonPathSanitizers { get; } = new();
 
+        /// <summary>
+        /// The list of JSON path sanitizers to use when sanitizing a JSON request or response body.
+        /// By default, "$..primaryKey", "$..secondaryKey", "$..primaryConnectionString", "$..secondaryConnectionString",
```

Comment:
Where is this actually defined? If it is local, should we point to that instead of listing them here so that this always is in sync?